docs/topics/mta-rn-upgrade-cli.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-rn-upgrade-cli_{context}"]
+= Setting the parameter in the CLI tool after upgrading to version {ProductVersion}
```

Comment:
MTA CLI cannot set rwx parameter .
Changes can be made only via TACKLE CR in openshift or in openshift console not in MTA CLI .

So this looks wrong.